ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml

```diff
@@ -3505,13 +3505,6 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-vsphere-upi-zones
-- as: vsphere-ipi-amd-f14-netobserv
```

Comment:
Why vsphere removed from 4.16?

Reply:
https://redhat.enterprise.slack.com/docs/T027F3GAJ/F06Q9BMS3HV refer this matrix, 4.17 is new qualifying release and will be replaced by 4.18 soon.